iele-gas.md

```diff
@@ -461,7 +472,7 @@ The bitwise expressions have a constant cost plus a linear factor in the number
 
 ```k
     rule #compute [ _ = addmod W0 , W1 , W2, SCHED ] => Gadd < SCHED > +Int maxInt(intSize(W0), intSize(W1)) *Int Gaddword < SCHED > +Int Cdiv(SCHED, maxInt(intSize(W0), intSize(W1)) +Int 1, intSize(W2))
-    rule #compute [ _ = mulmod W0 , W1 , W2, SCHED ] => Cmul(SCHED, intSize(W0), intSize(W1)) +Int Cdiv(SCHED, intSize(W0) +Int intSize(W1), intSize(W2)) +Int Gmulmod < SCHED >
```

Comment:
We could but then we would have to change the gas formulas for Cmul and Cdiv because otherwise the gas cost of mulmod would be unnecessarily high. So I'm inclined to leave it this way for now.